svc/wallet/service.go

```diff
@@ -811,3 +822,29 @@ func (s *Service) P2PTransfer(ctx context.Context, uid, recipientID uuid.UUID, a
 
 	return nil
 }
+
+// GetMultiplier returns multiplier according to wallet's stake level.
+func (s *Service) GetMultiplier(ctx context.Context, userID uuid.UUID) (_ int32, err error) {
+	stake, err := s.wr.GetStakeByUserID(ctx, userID)
+	if err != nil {
+		// since multiplier == 1 equals to no multiplier should return 1
+		return 1, fmt.Errorf("could not get stake by user id: %w", err)
+	}
+
+	stakeLevels, err := s.wr.GetAllStakeLevels(ctx)
+	if err != nil {
+		return 1, fmt.Errorf("could not get stake levels: %w", err)
+	}
+
+	for i := 0; i < len(stakeLevels); i++ {
+		if stakeLevels[i].Disabled.Bool {
+			continue
+		}
+
+		if stakeLevels[i].MinStakeAmount.Float64 < stake.StakeAmount && stakeLevels[i].MinDaysAmount.Int32 < stake.StakeDuration.Int32 {
+			return stakeLevels[i].Multiplier.Int32, nil
+		}
+	}
+
+	return 1, nil
```

Comment:
```suggestion
        return 0, nil
```
this method returns percent to add to the user`s reward, and if you return 1 by default, users always have +1% to rewards